.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - stable
```

Comment:
do we have a baseline node version we should we?  this might be something to consider at some point. otherwise we will run into issues where my older version of node does not have the same features used when running this in CI.  Not really something to address now, but something to keep in mind as we progress this.